kernel/setup/steps/ezstep_database_choice.php

```diff
@@ -78,7 +78,7 @@ function init()
             }
         }
 
-        if ( $databaseCount != 1 )
```

Comment:
Could you explain that change? Why was it necessary to invert this?

Reply:
In testing in the short term it was required, but only because the implementation was not whole. I have reverted this change.